### ISSUE TITLE
NSFE fade-out issues

### ISSUE THREAD

**Issue by BlackGuyRX** [droidmjt/Droidsound#558]
I can't quite figure out how to get NSFE files to fade out properly, they seem to fade out too early or short jingles fade even tho they shouldn't. It feels like I have zero control over anything and the settings are extremely confusing. I'm running v1.67.212 on Android 10

**Comment by droidmjt** [comment]
There is no fade-in at all.  What settings are confusing? Please send an example song what fades out too early.

**Comment by BlackGuyRX** [comment]
I'm so sorry, I meant fade out, just edited my post co clarify. Meant to respond hours ago but I was at work. Anyway, I've been listening to a lot of NSFEs on an older version on an old phone I used. I don't remember what exactly I did to tune in the settings to get what I wanted but after starting over on a new phone, I've had a hard time trying to get .nsfe files to fade after the appropriate loop (or just fade out too early before the second loop) or short songs and jingles that weren't intended to fade out yet they still do.
